{
  "specName": "cypress\\integration\\OrangeHRM\\PIM\\AddEmployee.feature",
  "title": "Add new Employee",
  "suiteName": "Add New",
  "testName": "Add New Add new Employee",
  "testError": "The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Network Error\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.",
  "testCommands": [
    "step **Login to OrangeHRM**",
    "visit https://opensource-demo.orangehrmlive.com/web/index.php/auth/login",
    "clearCookies ",
    "request ",
    "clearLocalStorage ",
    "visit https://opensource-demo.orangehrmlive.com/web/index.php/auth/login",
    "get input[name=\"username\"]",
    "request ",
    "type Admin",
    "get input[name=\"password\"]",
    "type admin123",
    "get button[type=\"submit\"]",
    "click ",
    "page load --waiting for new page to load--",
    "new url https://opensource-demo.orangehrmlive.com/web/index.php/dashboard/index",
    "request ",
    "step **I click on PIM in side panel**",
    "get span[class='oxd-text oxd-text--span oxd-main-menu-item--name']",
    "contains PIM",
    "click ",
    "request ",
    "request ",
    "page load --waiting for new page to load--",
    "request ",
    "request ",
    "request ",
    "request ",
    "request ",
    "uncaught exception AxiosError: Network Error"
  ]
}
